.github/workflows/trivy-analysis.yml

```diff
@@ -11,20 +12,20 @@ on:
 
 jobs:
   build:
-    name: Build
+    name: Run Trivy against OSS container
     runs-on: "ubuntu-18.04"
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
 
       - name: Build an image from Dockerfile
         run: |
-          IMAGE_TAG=docker.io/couchbase/observability-stack:${{ github.sha }} tools/build-oss-container.sh
+          IMAGE_TAG=docker.io/couchbaselabs/observability-stack:${{ github.sha }} tools/build-oss-container.sh
```

Comment:
Do we know what image name it'll be published at for 0.1? Because it might be worth getting it in now to avoid changing it all later.

Reply:
Sort of but it changes every two seconds depending on PM and others feelings...
We do need to make sure we're testing whatever is on the main branch and not just the version. Or do you mean just switching this to the name but leave the SHA there? We can't use the dockerhub one as the build team own the upload process for that.

Reply:
I'm just trying to avoid us having to change the name all over the place in the future - if we can pick one and stick to it I'm happy (maybe even one just for testing purposes). Just a nit anyway.

Reply:
Absolutely, I'm also trying a matrix build on the other one to see if we can just have one definition there.